app/models/concerns/attribute_displayable.rb

```diff
@@ -0,0 +1,13 @@
+module AttributeDisplayable
```

Comment:
AttributeDisplayable was kind of a shot in the dark, but this one pertains to convenience methods for displaying attributes on the patient model in particular. It was kind of a catchall nothing else really fit into here, at least prior to merging pregnancy in.

I added a comment to indicate that it displays attributes on the patient model, but this really could just as easily be `PhoneDisplayable`. Thoughts?